qcut/docs/task/log_v7.md

```
@@ -0,0 +1,261 @@
+Download the React DevTools for a better development experience: https://reactjs.org/link/react-devtoolsYou might need to use a local HTTP server (instead of file://): https://reactjs.org/link/react-devtools-faq
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)